server/routerlicious/packages/lambdas/src/deli/checkpointContext.ts

```diff
@@ -152,9 +164,7 @@ export class CheckpointContext {
 				getLumberBaseProperties(this.id, this.tenantId),
 				error,
 			);
-			return new Promise<void>((resolve, reject) => {
-				resolve(this.checkpointCore(checkpoint));
-			});
+			throw error;
```

Comment:
Above this (can't comment on it because it's too far from the "diff", there is a comment saying `// Retry the checkpoint on error`. I think that is not relevant in this code block now because the error is retried internally. Can we delete that comment to avoid confusion?

Reply:
Removed the outdated comment